src/stylesheets/project_specific/developer/_developer-settings.scss

```diff
@@ -4,4 +4,19 @@
     margin-right: 12px;
   }
 
+  .plan .free {
```

Comment:
Ordinarily, I'd agree with that suggestion.  But `plan`, `free`, and `metered` are all tightly coupled to both the rails app and the javascript used for toggling the spending limit field.  Adding classes (which are name spaced to this section of the developer app anyway) is preferable to making updates to the app/js logic at this point.